src/components/ColorPicker/ColorPreview.tsx

```diff
@@ -15,7 +15,7 @@ export const ColorPreview: FC<ColorPreviewProps> = ({ color }) => {
 
     return (
         <div
-            className="tw-sticky tw-box-content tw-shrink-0 tw-h-2 tw-top-0 tw-bg-white tw-z-20 dark:tw-bg-black-95"
+            className="tw-sticky tw-box-content tw-shrink-0 tw-h-2 tw-top-0 tw-bg-white tw-z-20 dark:tw-bg-black-95 tw-border-t tw-mt-[-1px] tw-border-l tw-border-r tw-border-line tw-rounded-t"
```

Comment:
`tw-border-t tw-mt-[-1px] tw-border-l tw-border-r` could be `tw-mt-[-1px] tw-border tw-border-b-0`
Maybe it would be easier to read, WDYT?